pandas/_libs/parsers.pyx

```diff
@@ -1384,6 +1384,7 @@ STR_NA_VALUES = {
     "nan",
     "-nan",
     "",
+    "None",
```

Comment:
This is necessary to convert "None" to boolean NA, not sure if we want to add a whatsnew?

Reply:
I think it would be good just to be safe. Just in case people were expecting "None" to come back as "None` normally?

Reply:
Added